### PR TITLE
[VEN-2349] Refactor networkAddresses

### PR DESCRIPTION
**Pull request by 0xlucian**
## Description

This PR refactors the networkAddresses file, leaving only addresses there used by the VIP framework. All other addresses are defined in each VIP itself.

Resolves VEN-2349

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,8 +18,11 @@
     "@morpho-labs/gnosis-tx-builder": "^1.3.1",
     "@openzeppelin/contracts": "^4.8.2",
     "@openzeppelin/contracts-upgradeable": "^4.8.2",
-    "@venusprotocol/isolated-pools": "2.3.0-dev.9",
-    "@venusprotocol/venus-protocol": "^6.1.0-dev.3",
+    "@venusprotocol/governance-contracts": "^1.4.0",
+    "@venusprotocol/isolated-pools": "^2.5.0",
+    "@venusprotocol/oracle": "^1.10.0-dev.2",
+    "@venusprotocol/token-bridge": "^1.1.0-dev.1",
+    "@venusprotocol/venus-protocol": "^7.3.0-dev.2",
```
Comment (chechu) [inline package.json:25]: I think we can update these versions to the last no-dev ones

**Comment by GitGuru7** [comment]
To avoid any human error, I would use addresses from networkAddresses files. We can use npm packages to fetch the latest addresses, This way we may avoid any miss-happening cause due to wrong address.

**Comment by chechu** [comment]
> To avoid any human error, I would use addresses from networkAddresses files. We can use npm packages to fetch the latest addresses, This way we may avoid any miss-happening cause due to wrong address.
> 
> cc: @chechu

Addresses can change in the npm packages across different versions. By hardcoding the specific addresses related to the VIP in the VIP files we ensure the simulations will work in the future, independently of the npm packages used.

On the other hand, the addresses used by the VIP framework should use the networkAddress file, that should import the addresses from the npm packages.

**Comment by GitGuru7** [comment]
> > To avoid any human error, I would use addresses from networkAddresses files. We can use npm packages to fetch the latest addresses, This way we may avoid any miss-happening cause due to wrong address.
> > cc: @chechu
> 
> Addresses can change in the npm packages across different versions. By hardcoding the specific addresses related to the VIP in the VIP files we ensure the simulations will work in the future, independently of the npm packages used.
> 
> On the other hand, the addresses used by the VIP framework should use the networkAddress file, that should import the addresses from the npm packages.

I agree, For the addresses which never going to be change like Proxies, governance etc. can be used from networkAddresses files.

File: multisig/simulations/vip-002/vip-002-sepolia/simulations.ts [hunk 2]
```diff
@@ -12,17 +12,37 @@ import { vip002 } from "../../../proposals/vip-002/vip-002-sepolia";
...
 
 const BLOCKS_PER_YEAR = 2_252_571; // assuming a block is mined every 14 seconds
 
+const COMPTROLLER_CORE = "0x7Aa39ab4BcA897F403425C9C6FDbd0f882Be0D70";
+const COMPTROLLER_STABLECOINS = "0x18eF8D2bee415b731C25662568dc1035001cEB2c";
+const COMPTROLLER_CURVE = "0xD298182D3ACb43e98e32757FF09C91F203e9E67E";
+const MOCK_WBTC = "0x92A2928f5634BEa89A195e7BeCF0f0FEEDAB885b";
+const MOCK_USDC = "0x772d68929655ce7234C8C94256526ddA66Ef641E";
+const MOCK_USDT = "0x8d412FD0bc5d826615065B931171Eed10F5AF266";
+const MOCK_WETH = "0x700868CAbb60e90d77B6588ce072d9859ec8E281";
```
Comment (chechu) [inline multisig/simulations/vip-002/vip-002-sepolia/simulations.ts:31]: Replace with the real WETH token we are using

File: multisig/simulations/vip-004/vip-004-opbnbtestnet/simulations.ts [hunk 3]
```diff
@@ -58,7 +57,7 @@ forking(16716512, () => {
     });
 
     it("Should set correct reward token address", async () => {
-      const isActive = await xvsStore.rewardTokens(XVS);
+      const isActive = await xvsStore.rewardTokens(opbnbtestnet.XVS);
```
Comment (chechu) [inline multisig/simulations/vip-004/vip-004-opbnbtestnet/simulations.ts:60]: There are more uses of `XVS` in this file, to be replaced with `opbnbtestnet.XVS`. After doing that, we could remove the constant `XVS`

File: multisig/simulations/vip-004/vip-004-opbnbtestnet/simulations.ts [hunk 4]
```diff
@@ -14,7 +14,6 @@ import XVS_VAULT_ABI from "./abi/xvsvault.json";
 
 const { opbnbtestnet } = NETWORK_ADDRESSES;
 
-const XVS_VAULT_PROXY = "0xB14A0e72C5C202139F78963C9e89252c1ad16f01";
 const XVS_STORE = "0x06473fB3f7bF11e2E8EfEcC95aC55ABEFCb2e0A0";
 const XVS = "0xc2931B1fEa69b6D6dA65a50363A8D75d285e4da9";
```
Comment (chechu) [inline multisig/simulations/vip-004/vip-004-opbnbtestnet/simulations.ts:18]: To be removed

**Pull request closed without merging**